ovs-service/src/main/java/org/dcsa/ovs/mapping/TransportCallMapper.java

```diff
@@ -79,4 +81,30 @@ PortTerminalLocation mapPortTerminalLocation(TransportCall transportCall) {
 
     return portTerminalLocation;
   }
+
+  @Mappings(
+    value = {
+      @Mapping(target = "location", source = "location"),
+      @Mapping(target = "importVoyage.carrierVoyageNumber", source = "carrierImportVoyageNumber"),
+      @Mapping(target = "exportVoyage.carrierVoyageNumber", source = "carrierExportVoyageNumber"),
+      @Mapping(
+        target = "importVoyage.universalVoyageReference",
+        source = "universalImportVoyageReference"),
+      @Mapping(
+        target = "exportVoyage.universalVoyageReference",
+        source = "universalExportVoyageReference"),
+      @Mapping(target = "portCallStatusCode", source = "statusCode")
+    })
+  public abstract TransportCall toEntity(TransportCallTO transportCallTO);
+
+  LocationEntity mapLocation(PortTerminalLocation location) {
```

Comment:
There are 4 location types, correct? Is some mapping missing?

Reply:
Will add it.